tools/openapi-generator-maven-plugin/src/main/java/org/openmrs/plugin/OpenAPIMojo.java

```
@@ -0,0 +1,355 @@
+            }
+        }
+        
+    } catch (Exception e) {
+        getLog().warn(" Basic method calls failed: " + e.getMessage());
+    }
+    }    private void testGetRepresentationDescription(Object instance) {
+    try {
+        getLog().info("=== Testing getRepresentationDescription Method ===");
+        
+        // First, verify RestConstants is still accessible in this context
+        testRestConstantsInMethodContext();
+        
+        // Load representation classes
+        Class<?> representationClass = classLoader.loadClass("org.openmrs.module.webservices.rest.web.representation.Representation");
+        Class<?> defaultRepClass = classLoader.loadClass("org.openmrs.module.webservices.rest.web.representation.DefaultRepresentation");
+        Class<?> fullRepClass = classLoader.loadClass("org.openmrs.module.webservices.rest.web.representation.FullRepresentation");
+        
+        // Create representation instances
+        Object defaultRep = defaultRepClass.getDeclaredConstructor().newInstance();
+        Object fullRep = fullRepClass.getDeclaredConstructor().newInstance();
+        
+        // Get the method
+        Method method = instance.getClass().getMethod("getRepresentationDescription", representationClass);
+        
+        // Test DEFAULT representation
+        Object defaultResult = method.invoke(instance, defaultRep);
+        getLog().info(" DEFAULT result: " + defaultResult);
+        
+        // Test FULL representation  
+        Object fullResult = method.invoke(instance, fullRep);
+        getLog().info(" FULL result: " + fullResult);
+
+        if (defaultResult != null) {
+        extractSchemaFromDescription(defaultResult, "DEFAULT");
+        }
+
+        if (fullResult != null) {
+        extractSchemaFromDescription(fullResult, "FULL");
+        }
+        
+    } catch (Exception e) {
+        getLog().error("Method invocation failed: " + e.getMessage());
```

Avoid this catch-and-log pattern. It hides the stack trace. If the error is unrecoverable, just let it be thrown without catching.

Also, try to avoid catching `Exception`, as it is too general. You might mask genuine coding errors like `IndexOutOfBoundsException`.